src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyFactory.java

```diff
@@ -223,6 +231,10 @@ private MessageConsumer newResponseStream(
             {
                 newStream = newResponse.apply(httpBeginFW);
             }
+            else
+            {
+                newStream = newResponse.apply(defaultHttpBeginExRO);
+            }
```

Comment:
Better to assign `httpBeginEx = defaultHttpBeginEx` then call `newStream = newResponse.apply(httpBeginEx)`.

Reply:
Reminder to apply this feedback too.